package.json

```diff
@@ -68,41 +67,30 @@
     "wreck": "^5.4.0"
   },
   "devDependencies": {
+    "angular": "^1.5.5",
```

Comment:
We use angular `1.3.7` so all the angular dependencies need to be changed to that. Also we should lock everything too.